README.md

```diff
@@ -187,9 +203,107 @@ vaults retrieve bafybeifr5njnrw67yyb2h2t7k6ukm3pml4fgphsxeurqcmgmeb7omc2vlq
 You can specify the file where the retrieved content will be save:
 
 ```bash
-vaults retrieve --output filename bafybeifr5njnrw67yyb2h2t7k6ukm3pml4fgphsxeurqcmgmeb7omc2vlq
+vaults retrieve --output [FILENAME] bafybeifr5njnrw67yyb2h2t7k6ukm3pml4fgphsxeurqcmgmeb7omc2vlq
+```
+
+### HTTP APIs
```

Comment:
added docs for HTTP APIs (I've been sharing a Notion doc, so i figured it'd best if it's just public)